src/models/aspera-desktop.model.ts

```diff
@@ -105,7 +123,7 @@ export class ActivityTracking {
    * @param event the event type.
    */
   handleWebSocketEvents(event: WebsocketEvents): void {
```

Comment:
Little concern using `event` when this is actually an enum.  Kind of confusing from a JS dev perspective. Not really an issue; but maybe a different name so we know its not a traditional `Event` would be helpful